src/Features/VisualBasicTest/Utils.vb

```diff
@@ -0,0 +1,341 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Runtime.CompilerServices
+Imports System.Text
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests
```

Comment:
Was this moved from somewhere?

Reply:
Yes.  Temporary copy as things move down.  Bring tracked as something to remove once it all moves down.